deployment/cdk/opensearch-service-migration/lib/service-stacks/reindex-from-snapshot-stack.ts

```diff
@@ -113,19 +122,53 @@ export class ReindexFromSnapshotStack extends MigrationServiceCore {
             servicePolicies.push(getSecretsPolicy);
         }
 
+        const volumes = [sharedLogFileSystem.asVolume()];
+        const mountPoints = [sharedLogFileSystem.asMountPoint()];
+
+        // Calculate the volume size based on the max shard size
+        // Have space for the snapshot and an unpacked copy, with buffer
+        const planningSize = props.maxShardSizeGiB ?? 80;
+        const volumeSize = Math.max(
+            Math.ceil(planningSize * 2 * 1.15),
+            125 // Minimum volume size for ST1
+        )
+
+        // Volume we'll use to download and unpack the snapshot
+        const snapshotVolume = new ServiceManagedVolume(this, 'SnapshotVolume', {
```

Comment:
Is encryption an option?
<img width="681" alt="image" src="https://github.com/user-attachments/assets/ba911fe6-7059-4fe3-a9aa-bd9e4bf233a3">

Reply:
Yes, it is an option; I'm baffled why the default is "False".  Another good catch!

Reply:
Looks like the default is the ECS Cluster settings which we don't set

<img width="966" alt="image" src="https://github.com/user-attachments/assets/5827c2be-8e91-4be0-8d77-bca4f8343496">